jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -84,6 +84,33 @@ YUI.add('juju-env-go', function(Y) {
     return newObj;
   };
 
+  // Define the special machine scope used to specify unit placements.
+  var MACHINE_SCOPE = '#';
```

Comment:
Will it ever get used anywhere else? 

I believe we can use:

`const MACHINE_SCOPE = '#'`